spring-boot/src/test/java/org/springframework/boot/logging/LoggingApplicationListenerTests.java

```diff
@@ -279,6 +280,23 @@ public void parseLevelsCaseInsensitive() throws Exception {
 	}
 
 	@Test
+	public void parseLevelsCaseInsensitivePassesTurkeyTest() throws Exception {
+		Locale original = Locale.getDefault();
```

Comment:
`<shamelessPlug />`
I am developing a set of comon test annotions/rules.

Would it make sense to even start using that in Spring Boot test code?
https://github.com/Shredder121/testannotations/blob/213cc6232f3b899881bdf52c0e2ab09532a2cfc3/src/test/java/com/github/shredder121/testannotations/locale/LocaleRuleTest.java#L65-L70